Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBServiceTests/SyncPerformanceIntTestHarness.swift

```diff
@@ -0,0 +1,363 @@
+            }
+            stdDev = sqrt(sumOfSquared / Double(newData.count))
+        } else {
+            mean = 0.0
+            median = 0.0
+            max = 0.0
+            min = 0.0
+            stdDev = 0.0
+        }
+    }
+
+    lazy var asBson: Document = [
+        "mean": mean,
+        "median": median,
+        "max": max,
+        "min": min,
+        "stdDev": stdDev
+    ]
+}
+
+internal class IterationResult {
+    let time: Double
+    let networkSentBytes: Double
+    let networkReceivedBytes: Double
+    let cpu: Double
+    let memory: Double
+    let disk: Double
+    let threads: Double
+
+    init(time: Double, networkSentBytes: Double, networkReceivedBytes: Double,
+         cpu: Double, memory: Double, disk: Double, threads: Double) {
+        self.time = time
+        self.networkSentBytes = networkSentBytes
+        self.networkReceivedBytes = networkReceivedBytes
+        self.cpu = cpu
+        self.memory = memory
+        self.disk = disk
+        self.threads = threads
+    }
+}
+
+private class RunResults {
+    let numDocs: Int
+    let docSize: Int
+    let numOutliers: Int
+    let time: DataBlock
+    let networkSentBytes: DataBlock
+    let networkReceivedBytes: DataBlock
+    let cpu: DataBlock
+    let memory: DataBlock
+    let disk: DataBlock
+    let diskEfficiencyRatio: DataBlock
+    let threads: DataBlock
+
+    init(numDocs: Int,
+         docSize: Int,
+         numOutliers: Int,
+         time: [Double],
+         networkSentBytes: [Double],
+         networkReceivedBytes: [Double],
+         cpu: [Double],
+         memory: [Double],
+         disk: [Double],
+         threads: [Double]) {
+        self.numDocs = numDocs
+        self.docSize = docSize
+        self.numOutliers = numOutliers
+        self.time = DataBlock(data: time, numOutliers: numOutliers)
+        self.networkSentBytes = DataBlock(data: networkSentBytes, numOutliers: numOutliers)
+        self.networkReceivedBytes = DataBlock(data: networkReceivedBytes, numOutliers: numOutliers)
+        self.cpu = DataBlock(data: cpu, numOutliers: numOutliers)
+        self.memory = DataBlock(data: memory, numOutliers: numOutliers)
+        self.disk = DataBlock(data: disk, numOutliers: numOutliers)
+        self.threads = DataBlock(data: threads, numOutliers: numOutliers)
+
+        let diskEfficiencyArr = disk.map({$0 / Double((docSize + 12) * numDocs)})
+        self.diskEfficiencyRatio = DataBlock(data: diskEfficiencyArr, numOutliers: numOutliers)
+
+    }
+
+    lazy var asBson: Document = [
+        "numDocs": numDocs,
+        "docSize": docSize,
+        "status": "success",
+        "timeMs": time.asBson,
+        "networkSentBytes": networkSentBytes.asBson,
+        "networkReceivedBytes": networkReceivedBytes.asBson,
+        "cpu": cpu.asBson,
+        "memoryBytes": memory.asBson,
+        "diskBytes": disk.asBson,
+        "diskEfficiencyRatio": diskEfficiencyRatio.asBson,
+        "threads": threads.asBson
```

Comment:
[naming] seems like Jason (or someone) changed this on Android to "activeThreadCounts". We should make sure the Android and iOS param names match so there's a common data model between the two

Reply:
T'was me. `activeThreadCounts` is a bit more descriptive than "threads" 😄